src/scripts/main.js

```diff
@@ -1,3 +1,286 @@
 'use strict';
 
-// write your code here
+const tableGame = document.querySelector('.game-field');
+const rows = tableGame.rows;
+const button = document.querySelector('button');
+const score = document.querySelector('.game-score');
+
+const messageStart = document.querySelector('.message-start');
+const messageWin = document.querySelector('.message-win');
+const messageLose = document.querySelector('.message-lose');
+const cells = document.querySelectorAll('.field-cell');
+const column = 4;
```

Comment:
better to name is such a way that it contains more precise meaning like columnsCount or columnsNumber. Also most likely that in this app is should always equal to tableGame.rows.length